configuration/configuration.go

```diff
@@ -308,9 +310,11 @@ func (c *Data) GetTemplateValues() (map[string]string, error) {
 	}
 
 	return map[string]string{
-		"RECOMMENDER_EXTERNAL_NAME": c.v.GetString(varTemplateRecommenderExternalName),
-		"RECOMMENDER_API_TOKEN":     base64.StdEncoding.EncodeToString([]byte(c.v.GetString(varTemplateRecommenderAPIToken))),
-		"DOMAIN":                    c.v.GetString(varTemplateDomain),
+		"RECOMMENDER_EXTERNAL_NAME":      c.v.GetString(varTemplateRecommenderExternalName),
+		"RECOMMENDER_API_TOKEN":          base64.StdEncoding.EncodeToString([]byte(c.v.GetString(varTemplateRecommenderAPIToken))),
+		"CHE_KEYCLOAK_AUTH__SERVER__URL": c.v.GetString(varKeycloakAuthServerURL),
```

Comment:
is there a reason why this variable and the KEYCLOAK_CLIENT__ID have double underscores?

Reply:
those are the env vars that Che seems to want https://github.com/fabric8-services/fabric8-tenant-che/pull/33/commits/484191ea7df89d091fbaeec51073ef1899fd27f2